packs/data/feats.db/skill-mastery-investigator.json

```diff
@@ -27,7 +27,277 @@
+                        ],
+                        "value": "nat"
+                    },
+                    {
+                        "label": "PF2E.SkillOcc",
+                        "predicate": [
+                            "skill:occ:rank:1"
+                        ],
+                        "value": "occ"
+                    },
+                    {
+                        "label": "PF2E.SkillPrf",
+                        "predicate": [
+                            "skill:prf:rank:1"
+                        ],
+                        "value": "prf"
+                    },
+                    {
+                        "label": "PF2E.SkillRel",
+                        "predicate": [
+                            "skill:rel:rank:1"
+                        ],
+                        "value": "rel"
+                    },
+                    {
+                        "label": "PF2E.SkillSoc",
+                        "predicate": [
+                            "skill:soc:rank:1"
+                        ],
+                        "value": "soc"
+                    },
+                    {
+                        "label": "PF2E.SkillSte",
+                        "predicate": [
+                            "skill:ste:rank:1"
+                        ],
+                        "value": "ste"
+                    },
+                    {
+                        "label": "PF2E.SkillSur",
+                        "predicate": [
+                            "skill:sur:rank:1"
+                        ],
+                        "value": "sur"
+                    },
+                    {
+                        "label": "PF2E.SkillThi",
+                        "predicate": [
+                            "skill:thi:rank:1"
+                        ],
+                        "value": "thi"
+                    }
+                ],
+                "flag": "skillExpert",
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.SkillMastery.ExpertPrompt"
+            },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "upgrade",
+                "path": "system.skills.{item|flags.pf2e.rulesSelections.skillExpert}.rank",
+                "value": 2
+            },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "upgrade",
+                "path": "system.skills.{item|flags.pf2e.rulesSelections.skillMaster}.rank",
+                "value": 3
+            },
+            {
+                "adjustName": false,
+                "allowedDrops": {
+                    "label": "PF2E.SpecificRule.SkillMastery.AllowedDrops",
+                    "predicate": [
+                        "item:trait:skill"
+                    ]
+                },
+                "choices": {
+                    "query": "{\"system.traits.value\":{\"$elemMatch\":\"skill\"}}"
+                },
+                "flag": "skillMasteryRogue",
```

Comment:
Flag should probably be skillMasteryInvestigator

Reply:
Good catch, fixed